charts/misc-apps/values.yaml

```diff
@@ -96,3 +96,22 @@ chartmuseum:
   # chartmuseum.values -- Helm values
   # @default -- [upstream values](https://github.com/chartmuseum/charts/blob/main/src/chartmuseum/values.yaml)
   values: {}
+
+# prom2teams -- [prom2teams](https://github.com/idealista/prom2teams) ([example](./example/prom2teams.yaml))
+# @default -- -
+prom2teams:
+  # prom2teams.enabled -- Enable prom2teams
+  enabled: false
+  name: prom2teams
+  destination:
+    # prom2teams.destination.namespace -- Namespace
+    namespace: "infra-prom2teams"
+  # prom2teams.repoURL -- [repo](https://prometheus-msteams.github.io/prometheus-msteams)
+  repoURL: https://prometheus-msteams.github.io/prometheus-msteams
+  # prom2teams.chart -- Chart
+  chart: prom2teams
+  # prom2teams.targetRevision -- [prom2teams Helm chart](https://github.com/idealista/prom2teams/tree/master/helm)
+  targetRevision: "3.2.*"
```

Comment:
The chart from https://prometheus-msteams.github.io/prometheus-msteams  seems to be for deploying this prom2teams: https://github.com/prometheus-msteams/prometheus-msteams

But version 3.2.3 from https://github.com/idealista/prom2teams/ is seemingly a different implemetation of prom2teams

Which version do we intend to deploy? Can we compare the two versions to make an informed decision?

In any case the PR needs to be updated to only use on of them.

Reply:
Thanks for noticing, I just updated it.